### PR TITLE
steampipePackages.steampipe-plugins-aws: init at 0.132.0

### PR DESCRIPTION
**Pull request by anthonyroussel**
## Description of changes

* https://github.com/turbot/steampipe-plugin-aws
* https://github.com/turbot/steampipe-plugin-github

steampipePackages.steampipe-postgres-fdw is also coming, but looks like it is very difficult to compile and package with PostgreSQL. Maybe it could be a resolution to https://github.com/NixOS/nixpkgs/issues/215945

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by anthonyroussel** [comment]
@ofborg build steampipePackages

**Comment by weitzj** [comment]
@anthonyroussel  I would love to help you approve this. Can I do anything? Add myself to maintainers?

**Comment by weitzj** [comment]
Result of `nixpkgs-review pr 299274` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>steampipePackages.steampipe-plugin-aws</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>steampipePackages.steampipe-plugin-github</li>
  </ul>
</details>

**Comment by weitzj** [comment]
Result of `nixpkgs-review pr 299274` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>steampipePackages.steampipe-plugin-aws</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>steampipePackages.steampipe-plugin-github</li>
  </ul>
</details>

**Comment by weitzj** [comment]
Result of `nixpkgs-review pr 299274` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>steampipePackages.steampipe-plugin-aws</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>steampipePackages.steampipe-plugin-github</li>
  </ul>
</details>

**Comment by anthonyroussel** [comment]
> Result of `nixpkgs-review pr 299274` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
> 
> 1 package failed to build:
> * steampipePackages.steampipe-plugin-aws
> 
> 1 package built:
> * steampipePackages.steampipe-plugin-github

@weitzj Hello, thanks for the review!

It looks like the build of the `steampipePackages.steampipe-plugin-aws` package failed on your machine from the nixpkgs-review output you posted.

Did you get any errors when building this package?

**Comment by anthonyroussel** [comment]
@ofborg build

**Comment by weitzj** [comment]
I will check again tonight or the next says

**Comment by weitzj** [comment]
Result of `nixpkgs-review pr 299274` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>steampipePackages.steampipe-plugin-aws</li>
    <li>steampipePackages.steampipe-plugin-github</li>
  </ul>
</details>

**Comment by weitzj** [comment]
Looks good @anthonyroussel

File: pkgs/tools/misc/steampipe-packages/steampipe-plugin-aws/default.nix [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    "-w"
+  ];
+
+  doCheck = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out
+    cp $GOPATH/bin/${pname} $out/${pname}.plugin
```
[the remark this answers is not in the document]
Comment (anthonyroussel) [reply pkgs/tools/misc/steampipe-packages/steampipe-plugin-aws/default.nix]: Thanks for the review! I replaced pname by steampipe-plugin-aws.

**Pull request merged**